### DISCUSSION TITLE
VS Code Copilot Activation Failed

### DISCUSSION THREAD

**Discussion by benliu4** [community/community#18121] [Copilot Conversations]
Hey Guys,
I got off the Copilot waitlist, and am currently trying to get started using it in VS Code. I have it installed, and am signed into my Github account, however I don't receive any of the reccomendations. Looking in the bottom right corner, I get a message that says Copilot activation failed. And also the message "Extension activation failed, run the 'Developer: Toggle Developer Tools' command for more information."

It also says "Please upgrade your Copilot extension to continue using this service."

Anyone know what might be wrong?

![image](https://user-images.githubusercontent.com/96307509/172753249-d174af95-8594-43ba-b813-6e75b1f3736e.png)

**Comment by tamalCodes** [comment]
This does seems like an installation bug, uninstall it, logout. 
The re-login and install it again. 

This worked for me !
